docs/dev/app/listings.md

```diff
@@ -1,9 +1,388 @@
+- `options`: All options for a given filter that a user can select from (derived from the listing data). This can be a map or a set. A map is used when you want the options' display-text to be different from the stored-text. For the map, keys are the actual values stored, and values are the displayed value
+- `placeholder`: The placeholder display text on the filter dropdown when no values of the current filter are selected. 
+- `value`: The selected values of the filter. This should be tracked as a state in the main component, and its corresponding setState function should be passed into `onChange` below.
+- `onChange`: Usually of the form `newValue => setFieldFilters(newValue)`. This function is called when the values of the filter dropdown is changed, i.e. when the user selects/deselects a value of a the current filter. 
+- (optional) `fullText`: A string that is the default display name of the filter dropdown button. Currenlty, this is only used for the "Remote/In Person" filter. 
+
+### `ListingCard` 
+- ListingCard renders each listing card, containing the condensed listing information.
+
+**ListingCard Props**
+- `listing`: *Listing*. The listing whose ListingCard will be rendered. The rendering of fields is dependent on the `listing_type` of `listing`. 
+- (optional) `isSelected`: *boolean*. Default value: false. If true, the card will be highlighted. 
+- (optional) `onClick`: *(id: UUID) => void*. An additional event handler triggered when the ListingCard is clicked. 
+- (optional) `interpretation`: *boolean*. Default value: false. Should be `true` for Langauge Support listings. Used to distinguish between Case Interpretations and Cases. 
+
+**Example Usage**
+- Langauge Support listings: `<ListingCard listing={listingData} isSelected={isSelected} interpretation />`
+- Non-Langauge Support listings: `<ListingCard listing={listingData} isSelected={isSelected} />`
+
+### `ListingDetails`
+- ListingDetails displays the details of a selected listing. If the user is logged in and onboarded, the ProfileMatch and InterestForm appear. Otherwise, the Profilematch and InterestForm will be replaced by buttons directing the user to log in and complete onboarding.
+- To display the fields underneath "Highlights", an instance of the `ListingFields` component is used. For nullable fields, if a listing field is null/undefined, the field will appear as "Not Available." However, some fields are non-nullable.
+
+**ListingDetails Props**
+- `listingData`: *Listing*. The listing whose ListingDetails will be rendered. The rendering of fields is dependent on the `listing_type` of `listingData`. 
+- (optional) `interpretation`: *boolean*. Default value: false. Should be `true` for Langauge Support listings. Used to distinguish between Case Interpretations and Cases. 
+
+**Example Usage**
+- Langauge Support listings: `<ListingDetails listingData={listingData} interpretation />`
+- Non-Langauge Support listings: `<ListingDetails listingData={listingData} />`
```

Comment:
if we update the previous usage, it could be good to update this as well to be consistent

Reply:
sg, i've updated all Example Usage lines to be code blocks